### PR TITLE
CallTarget Integration: ADO.NET MySqlData client version 6

### PR DESCRIPTION
**Pull request by tonyredondo**
Add MySqlData client version 6 support and execute the tests in multiple client versions.

@DataDog/apm-dotnet

### CODE REVIEW

File: test/Datadog.Trace.ClrProfiler.IntegrationTests/AdoNet/MySqlCommandTests.cs [hunk 1]
```diff
@@ -31,6 +103,10 @@ public void SubmitsTracesWithNetStandard(bool enableCallTarget, bool enableInlin
             var expectedSpanCount = 50; // 7 queries * 7 groups + 1 internal query
 #else
             var expectedSpanCount = 78; // 7 queries * 11 groups + 1 internal query
+            if (packageVersion == "6.8.8")
+            {
+                expectedSpanCount = 76; // For this version the callsite instrumentation returns 2 spans less.
```
Comment (kevingosse) [inline test/Datadog.Trace.ClrProfiler.IntegrationTests/AdoNet/MySqlCommandTests.cs:108]: Out of curiosity, have you checked what are those 2 missing spans?

**Review by kevingosse — APPROVED** [review]

File: test/Datadog.Trace.ClrProfiler.IntegrationTests/AdoNet/MySqlCommandTests.cs (hunk 1 above)
Comment (tonyredondo) [reply test/Datadog.Trace.ClrProfiler.IntegrationTests/AdoNet/MySqlCommandTests.cs:108]: No, didn't check which are the 2 missing spans.

File: docker-compose.yml [hunk 2]
```diff
@@ -67,6 +67,16 @@ services:
     - MYSQL_PASSWORD=mysqldb
     ports:
     - "127.0.0.1:3307:3306"
+    
+  mysql57:
```
Comment (lucaspimentel) [inline docker-compose.yml:71]: The two MySql client versions can't access the same server?

**Review by lucaspimentel — APPROVED** [review]
LGTM. Left a question about the possibility of using a single MySql server to test both client versions.

File: docker-compose.yml (hunk 2 above)
Comment (tonyredondo) [reply docker-compose.yml:71]: Nope IICR they changed the login security protocol between versions. But I have to double check that

**Pull request merged**